src/renderer/hooks/useTestImport.ts

```
@@ -50,25 +50,26 @@ export const useTestImport = () => {
     setModalOpen: Dispatch<SetStateAction<boolean>>,
   ) {
     let data: TestDiscoverContainer;
-    if (settings.importType == "Python") {
+    if (settings.importType == "python") {
       data = {
         response: [{ testName: path, path: path }],
         missingLibraries: [],
       };
     } else {
-      const response = await baseClient.get("discover-pytest", {
-        params: {
-          path: path,
-          oneFile: settings.importAsOneRef,
-        },
-      });
-      data = JSON.parse(response.data);
+      data = await captain
```

Errors are not handled here.